src/fragments/lib-v1/auth/js/user_session/fetch_auth_session.mdx

```diff
@@ -0,0 +1,11 @@
+```ts
+async function handleFetchAuthSession() {
+	try {
+		const session = await fetchAuthSession();
+		console.log(`The user tokens: ${session.tokens}`);
+		console.log(`The user credentials: ${session.credentials}`);
```

Comment:
We might want to refactor this into a comment, e.g. `// Tokens & credentials available via session.tokens and session.credentials` or something to avoid recommending people console log sensitive info

```suggestion
		console.log(`The user tokens: ${session.tokens}`);
		console.log(`The user credentials: ${session.credentials}`);
```